gravitee-apim-console-webui/src/management/api/documentation-v4/components/documentation-new-page/documentation-new-page.component.html

```diff
@@ -59,31 +57,59 @@
                   <gio-card-content-title>Import from file</gio-card-content-title>
                 </gio-form-selection-inline-card-content>
               </gio-form-selection-inline-card>
-              <gio-form-selection-inline-card value="HTTP">
-                <gio-form-selection-inline-card-content icon="gio:language">
-                  <gio-card-content-title>Import from URL</gio-card-content-title>
+              <gio-form-selection-inline-card value="EXTERNAL">
+                <gio-form-selection-inline-card-content icon="gio:git-branch">
+                  <gio-card-content-title>Link to External Source</gio-card-content-title>
                 </gio-form-selection-inline-card-content>
               </gio-form-selection-inline-card>
             </gio-form-selection-inline>
+
+            @if (form.controls.sourceType.value === 'EXTERNAL') {
+              <div class="stepper__content__source">
+                <div class="mat-subtitle-2">Page source</div>
+
+                <gio-form-selection-inline class="gio-radio-group stepper__content__source" formControlName="source">
+                  @for (fetcher of fetchers; track fetcher.id) {
+                    <gio-form-selection-inline-card [value]="fetcher.id">
+                      <gio-form-selection-inline-card-content [img]="fetcher.iconPath">
+                        <gio-card-content-title>{{ fetcher.name }}</gio-card-content-title>
+                      </gio-form-selection-inline-card-content>
+                    </gio-form-selection-inline-card>
+                  }
+                </gio-form-selection-inline>
+              </div>
+            }
           </div>
           <div class="stepper__actions">
-            <button mat-flat-button color="primary" matStepperNext>Next</button>
+            <button mat-flat-button color="primary" matStepperNext [disabled]="form.controls.source.invalid">Next</button>
             <button mat-stroked-button matStepperPrevious>Previous</button>
           </div>
         </mat-step>
         <mat-step>
-          <ng-template matStepLabel>{{ source === 'IMPORT' ? 'Upload a file' : 'Add content' }}</ng-template>
+          <ng-template matStepLabel>
+            @switch (form.value.sourceType) {
+              @case ('FILL') {
+                {{ 'Add content' }}
```

Comment:
I believe that the moustaches + quotation marks can be removed for each of the cases:
```suggestion
                Add content
```